### PR TITLE
Hotfix/other user profile

### PR DESCRIPTION
**Pull request by vladkudinov89**
Refactor logic view user's reviews in OtherUserPage

### CODE REVIEW

File: frontend/src/components/users/ReviewsContainer.vue [hunk 1]
```diff
@@ -34,26 +34,34 @@ export default {
...
+    methods: {
+        ...mapActions({
+            fetchAllPlaces: 'users/fetchAllPlaces'
+        })
+    },
     created() {
-        this.$store.dispatch('place/fetchPlaces', this.$route.query);
+        this.$store.dispatch('users/fetchAllPlaces')
+            .then(()=>{
+                this.$parent.$data.isLoading = false;
```
Comment (lenchvolodymyr) [inline frontend/src/components/users/ReviewsContainer.vue]: it is not a good approach to change parent state, it is better to emit custom event "onLoad" and in the parent component do what you wish

File: frontend/src/components/users/ReviewsContainer.vue [hunk 2]
```diff
@@ -28,32 +28,42 @@ import ReviewsContainerItem from '@/components/users/ReviewsContainerItem';
...
         },
     },
+    methods: {
+        ...mapActions({
+            fetchAllPlaces: 'users/fetchAllPlaces'
+        })
+    },
     created() {
-        this.$store.dispatch('place/fetchPlaces', this.$route.query);
+        this.$store.dispatch('users/fetchAllPlaces')
```
Comment (nemoipaha) [inline frontend/src/components/users/ReviewsContainer.vue]: if you use ...mapActions, you can just call this.fetchAllPlaces() here

File: frontend/src/store/modules/users/actions.js [hunk 3]
```diff
@@ -12,5 +12,16 @@ export default {
                     reject(error.response.data.error);
                 });
         });
-    }
+    },
+    fetchAllPlaces:(context) => {
```
Comment (nemoipaha) [inline frontend/src/store/modules/users/actions.js]: not so good, use another api calls

File: frontend/src/store/modules/users/state.js [hunk 4]
```diff
@@ -7,5 +7,6 @@ export default {
         facebook_url: '',
         twitter_url: '',
         instagram_url: '',
-    }
+    },
+    places: []
```
Comment (nemoipaha) [inline frontend/src/store/modules/users/state.js]: state should have reviews: {} and profile only, lists will be fetched from userList module

**Pull request closed without merging**